subprojects/libsfdo.wrap

```diff
@@ -0,0 +1,3 @@
+[wrap-git]
+url = https://gitlab.freedesktop.org/vyivel/libsfdo.git
+revision = v0.1.2
```

Comment:
You might want to fetch the latest (at the moment) commit instead or wait for 0.1.3, the latest release is kinda buggy (due to me not writing tests sooner).

Reply:
Okay. Updated to `head`.